terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java

```diff
@@ -0,0 +1,203 @@
+        downloadPDFOutput.setAdultUnitPrice(75000);
+        downloadPDFOutput.setConductor("Yes");
+        downloadPDFOutput.setTourAbs(
+                "そこは別天地、静寂と湯けむりの待つ宿へ… 詳しい情報はお取り合わせをお願い致します。");
+        downloadPDFOutput.setCustomerAdd("千葉県八千代市上高野");
+        downloadPDFOutput.setCustomerJob("プログラマ");
+        downloadPDFOutput.setTourDays("7");
+        downloadPDFOutput.setCustomerName("木村　太郎");
+        downloadPDFOutput.setChildUnitPrice(37500);
+        downloadPDFOutput.setDepName("北海道");
+        downloadPDFOutput.setArrName("北海道");
+        downloadPDFOutput.setCustomerMail("tarou@example.com");
+        downloadPDFOutput.setAdultCount(5);
+        downloadPDFOutput.setCustomerCode("00000001");
+        downloadPDFOutput.setReserveNo("00000001");
+        downloadPDFOutput.setRemarks("特になし");
+        downloadPDFOutput.setAccomTel("018-123-4567");
+        downloadPDFOutput.setCustomerPost("276-0022");
+        downloadPDFOutput.setAdultPrice(375000);
+        downloadPDFOutput.setChildPrice(112500);
+        downloadPDFOutput.setSumPrice(487500);
+        downloadPDFOutput.setPaymentTimeLimit("2019/03/24");
+        downloadPDFOutput.setReservedDay(SDF.parse("2019/02/21"));
+        downloadPDFOutput.setDepDay(SDF.parse("2019/03/31"));
+        downloadPDFOutput.setCustomerBirth(SDF.parse("1975/01/05"));
+        downloadPDFOutput.setPrintDay(SDF.parse("2019/03/06"));
+
+        Map<String, Object> model = new HashMap<String, Object>();
+        model.put("downloadPDFOutput", downloadPDFOutput);
+        model.put("downloadPDFName", "reservationReport");
+
+        PdfStamper stamper = null;
+        stamper = new PdfStamper(new PdfReader(RESERVATION_REPORT_PDF_PLACE), new ByteArrayOutputStream(OUTPUT_BYTE_ARRAY_INITIAL_SIZE));
+
+        HttpServletRequest request = new MockHttpServletRequest();
+
+        HttpServletResponse response = new MockHttpServletResponse();
+
+        try {
+            reservationReportPdfStamperView.mergePdfDocument(model, stamper,
+                    request, response);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+
+        AcroFields form = stamper.getAcroFields();
+        assertThat(form.getField("referenceName"), is(
+                "TERASOLUNA TRAVEL CUSTOMER CENTER"));
+        AcroFields.Item referenceNameItem = form.getFieldItem("referenceName");
+        PdfDictionary referenceNameMerged = referenceNameItem.getMerged(0);
+        TextField referenceNameTextField = new TextField(null, null, null);
+        try {
+            form.decodeGenericDictionary(referenceNameMerged,
+                    referenceNameTextField);
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        } catch (DocumentException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
```

Comment:
- [ ] You can unit catch statements to catch super class (Exception) or multiple catch.

Reply:
8e72b2ada7ab00f10862e82068a9fa6e42f80a6a

I changed to throw `Exception` in this place .